config.py

```diff
@@ -99,6 +99,8 @@ class Config:
                                else None)
     # https://www.elastic.co/blog/index-vs-type
 
+    NYC_GOV_BASE = 'www1.nyc.gov'
```

Comment:
This should probably be with the other constants that have to do with default data.